src/main/java/it/reply/orchestrator/dto/cmdb/ChronosService.java

```diff
@@ -47,10 +47,11 @@ public ChronosService(
       @Nullable String region,
       @NonNull String hostname,
       @Nullable String parentServiceId,
-      @NonNull ChronosServiceProperties properties,
-      boolean iamEnabled) {
+      boolean iamEnabled,
+      @Nullable boolean publicIpAssignable,
```

Comment:
```suggestion
      boolean publicIpAssignable,
```

Reply:
In truth it is nullable even in the CloudService base class ... I defined it nullable to maintain compatibility with existing services (which do not have that setting)

Reply:
then it must be removed also from the base class

Reply:
I mean you missed it :D What about backward compatibility?

Reply:
you are committing changes without answering this question....